rulesdir does not seem to work correctly

**What version of textlint are you using?**
11.2.5

**What file type (Markdown, plain text, etc.) are you using?**
html

**What did you do? Please include the actual source code causing the issue.**
I'm trying to work on a new rule and want to test it. The docs say that if you specify --rulesdir, it will ignore "default" rules, which I assume are the rules specified in my .textlintrc.

**What did you expect to happen?**
I expected *only* my new rule to be used.

**What actually happened? Please include the actual, raw output from textlint.**

It ignored my new rule and ran the rules specified in .textlintrc. It is certainly possible something is wrong with my new rule, but I can't tell from the debug output why it seems to be ignored.

```
➜  textlinttest ./node_modules/.bin/textlint --rulesdir ./textlint-rule-must-cap ray.html --debug
  textlint:cli textlint --version: 11.2.5 +0ms
  textlint:cli Running on files, stdin-filename: undefined +2ms
  textlint:module-loader config Config {
  textlint:module-loader   configFile: '/Users/rcamden/projects/textlinttest/.textlintrc',
  textlint:module-loader   rulesBaseDirectory: undefined,
  textlint:module-loader   disabledRules: [],
  textlint:module-loader   rules: [ 'ng-word', 'spelling', 'terminology' ],
  textlint:module-loader   disabledFilterRules: [],
  textlint:module-loader   filterRules: [],
  textlint:module-loader   presets: [],
  textlint:module-loader   plugins: [ 'html' ],
  textlint:module-loader   pluginsConfig: { html: true },
  textlint:module-loader   rulesConfig:
  textlint:module-loader    { 'ng-word': { words: [] },
  textlint:module-loader      spelling: { language: 'en-us', skipPatterns: [Array] },
  textlint:module-loader      terminology: { defaultTerms: true } },
  textlint:module-loader   filterRulesConfig: {},
  textlint:module-loader   rulePaths: [ './textlint-rule-must-cap' ],
  textlint:module-loader   formatterName: undefined,
  textlint:module-loader   quiet: false,
  textlint:module-loader   color: true,
  textlint:module-loader   cache: false,
  textlint:module-loader   cacheLocation: '/Users/rcamden/projects/textlinttest/.textlintcache' } +0ms
  textlint:module-loader Loading rules from './textlint-rule-must-cap' +3ms
  textlint:module-loader Loading rules from /Users/rcamden/projects/textlinttest/node_modules/textlint-rule-ng-word/lib/ng-word.js +1ms
  textlint:module-loader Loading rules from /Users/rcamden/projects/textlinttest/node_modules/textlint-rule-spelling/lib/index.js +1ms
  textlint:module-loader Loading rules from /Users/rcamden/projects/textlinttest/node_modules/textlint-rule-terminology/index.js +30ms
  textlint:module-loader Loading rules from plugin: /Users/rcamden/projects/textlinttest/node_modules/textlint-plugin-html/lib/index.js +22ms
  textlint:engine-core Process files [ '/Users/rcamden/projects/textlinttest/ray.html' ] +0ms
  textlint:engine-core No Process files that are un-support extensions: [] +0ms
  textlint:kernel available extensions: [ '.htm', '.html', '.md', '.markdown', '.mdown', '.mkdn', '.mkd', '.mdwn', '.mkdown', '.ron', '.txt', '.text' ] +0ms
  textlint:kernel use plugin: html +1ms
  textlint:core-task spelling pushReport {"message":"cantt -> cant, canto, CAN'T, canst, cants, Cantu, Catt","index":32} +0ms
  textlint:core-task spelling pushReport {"message":"spelldfkdfslkld -> ","index":38} +54ms
  textlint:core-task spelling pushReport {"message":"samp -> champ, amp, camp, damp, lamp, ramp, tamp, vamp, same, sap, scamp, stamp, sump, swamp, Sam","index":64} +3ms
  textlint:core-task spelling pushReport {"message":"ve -> re, veg, vet, vex, be, e, he, me, we, ye, V, Va, Vb, Vi, Vs, Vt, ave, eve, vie, Ce, DE, Fe, Ge, IE, Le, Ne, OE, PE, Se, Te, VD, VF, VG, VJ, VP, Xe","index":93} +2ms
  textlint:cli lint results: [{"messages":[{"type":"lint","ruleId":"spelling","message":"cantt -> cant, canto, CAN'T, canst, cants, Cantu, Catt","index":32,"line":2,"column":29,"severity":2},{"type":"lint","ruleId":"spelling","message":"spelldfkdfslkld -> ","index":38,"line":2,"column":35,"severity":2},{"type":"lint","ruleId":"spelling","message":"samp -> champ, amp, camp, damp, lamp, ramp, tamp, vamp, same, sap, scamp, stamp, sump, swamp, Sam","index":64,"line":2,"column":61,"severity":2},{"type":"lint","ruleId":"spelling","message":"ve -> re, veg, vet, vex, be, e, he, me, we, ye, V, Va, Vb, Vi, Vs, Vt, ave, eve, vie, Ce, DE, Fe, Ge, IE, Le, Ne, OE, PE, Se, Te, VD, VF, VG, VJ, VP, Xe","index":93,"line":3,"column":7,"severity":2}],"filePath":"/Users/rcamden/projects/textlinttest/ray.html"}] +452ms
  textlint:@textlint/linter-formatter formatterName: stylish +0ms

/Users/rcamden/projects/textlinttest/ray.html
  2:29  error  cantt -> cant, canto, CAN'T, canst, cants, Cantu, Catt                                                                                                    spelling
  2:35  error  spelldfkdfslkld ->                                                                                                                                        spelling
  2:61  error  samp -> champ, amp, camp, damp, lamp, ramp, tamp, vamp, same, sap, scamp, stamp, sump, swamp, Sam                                                         spelling
  3:7   error  ve -> re, veg, vet, vex, be, e, he, me, we, ye, V, Va, Vb, Vi, Vs, Vt, ave, eve, vie, Ce, DE, Fe, Ge, IE, Le, Ne, OE, PE, Se, Te, VD, VF, VG, VJ, VP, Xe  spelling

✖ 4 problems (4 errors, 0 warnings)
```

--- a/README.md
+++ b/README.md
@@ -89,43 +89,44 @@ Your `.textlintrc` is a great way to maintain your rules.
 Run `textlint -h` for information on how to use the CLI.
 
 ```
-$ textlint -h
-textlint [options] file.md [file|dir|glob*]
+$ textlint [options] file.md [file|dir|glob*]
+  
+  Options:
+    -h, --help                 Show help.
+    -c, --config path::String  Use configuration from this file or sharable config.
+    --init                     Create the config file if not existed. - default: false
+    --fix                      Automatically fix problems
+    --dry-run                  Enable dry-run mode for --fix. Only show result, don't change the file.
+    --debug                    Outputs debugging information
+    -v, --version              Outputs the version number.
+  
+  Using stdin:
+    --stdin                    Lint text provided on <STDIN>. - default: false
+    --stdin-filename String    Specify filename to process STDIN as
+  
+  Output:
+    -o, --output-file path::String  Enable report to be written to a file.
+    -f, --format String        Use a specific output format.
+                               Available formatter          : checkstyle, compact, jslint-xml, json, junit, pretty-error, stylish, table, tap, unix
+                               Available formatter for --fix: compats, diff, json, stylish
+    --no-color                 Disable color in piped output.
+    --quiet                    Report errors only. - default: false
+  
+  Specifying rules and plugins:
+    --no-textlintrc            Disable .textlintrc
+    --plugin [String]          Set plugin package name
+    --rule [path::String]      Set rule package name
+    --preset [path::String]    Set preset package name and load rules from preset package.
+    --rulesdir [path::String]  Set rules from this directory and set all default rules to off.
+  
+  Caching:
+    --cache                    Only check changed files - default: false
+    --cache-location path::String  Path to the cache file or directory
+  
+  Experimental:
+    --experimental             Enable experimental flag.Some feature use on experimental.
+    --rules-base-directory path::String  Set module base directory. textlint load modules(rules/presets/plugins) from the base directory.
 
-Options:
-  -h, --help                 Show help.
-  -c, --config path::String  Use configuration from this file or sharable config.
-  --init                     Create the config file if not existed. - default: false
-  --fix                      Automatically fix problems
-  --dry-run                  Enable dry-run mode for --fix. Only show result, don't change the file.
-  --debug                    Outputs debugging information
-  -v, --version              Outputs the version number.
-
-Using stdin:
-  --stdin                    Lint text provided on <STDIN>. - default: false
-  --stdin-filename String    Specify filename to process STDIN as
-
-Output:
-  -o, --output-file path::String  Enable report to be written to a file.
-  -f, --format String        Use a specific output format.
-                             Available formatter          : checkstyle, compact, jslint-xml, json, junit, pretty-error, stylish, table, tap, unix
-                             Available formatter for fixer: compats, diff, json, stylish
-  --no-color                 Disable color in piped output.
-  --quiet                    Report errors only. - default: false
-
-Specifying rules and plugins:
-  --plugin [String]          Set plugin package name
-  --rule [path::String]      Set rule package name
-  --preset [path::String]    Set preset package name and load rules from preset package.
-  --rulesdir [path::String]  Set rules from this directory and set all default rules to off.
-
-Caching:
-  --cache                    Only check changed files - default: false
-  --cache-location path::String  Path to the cache file or directory
-
-Experimental:
-  --experimental             Enable experimental flag.Some feature use on experimental.
-  --rules-base-directory path::String  Set module base directory. textlint load modules(rules/presets/plugins) from the base directory.
 ```
 
 When running texlint, you can target files to lint using the glob patterns. Make sure that you enclose any glob parameter you pass in quotes.

--- a/package.json
+++ b/package.json
@@ -20,7 +20,6 @@
     "release": "lerna publish from-package",
     "release:canary": "lerna publish --canary",
     "collect-changelog": "git tag --points-at HEAD | xargs -I{} monorepo-utils-collect-changelog {} | pbcopy",
-    "pretest": "npm-run-all clean build",
     "test": "npm-run-all --parallel lint test:packages",
     "test:all": "npm-run-all test test:examples test:integration test:docs",
     "test:docs": "lerna run test --scope textlint-website",

--- a/packages/textlint/src/options.ts
+++ b/packages/textlint/src/options.ts
@@ -118,7 +118,7 @@ export const options = optionator({
             type: "Boolean",
             default: "true",
             description: "Disable .textlintrc",
-            example: "--no-textlintrc"
+            example: "textlint --no-textlintrc --rule textlint-no-todo README.md"
         },
         {
             option: "plugin",
@@ -128,18 +128,18 @@ export const options = optionator({
         },
         {
             option: "rule",
-            type: "[path::String]",
+            type: "[String]",
             description: "Set rule package name"
         },
         {
             option: "preset",
-            type: "[path::String]",
+            type: "[String]",
             description: "Set preset package name and load rules from preset package."
         },
         {
             option: "rulesdir",
             type: "[path::String]",
-            description: "Set rules from this directory and set all default rules to off."
+            description: "Use additional rules from this directory"
         },
         {
             heading: "Caching"

